mod_ci/controllers.py

```diff
@@ -429,7 +429,8 @@ def create_instance(compute, project, zone, test, reportURL) -> Dict:
                                            'ci-linux', 'startup-script.sh'), 'r').read()
         metadata_items = [
             {'key': 'startup-script', 'value': startup_script},
-            {'key': 'reportURL', 'value': reportURL}
+            {'key': 'reportURL', 'value': reportURL},
+            {'key': 'bucket', 'value': config.get('GCS_BUCKET_NAME', '')}
```

Comment:
I think we should not stop the execution of the application, we should just check and log the missing values of the config file at the start of the application. 
(Saying this because when I started the application first time, at that time I didn't had much knowledge of what to fill in the KVM-related fields, I guess the same goes for new users with GCP-related information).

Reply:
Hmm, definitely a better idea. What do you think about logging the functionalities that might not work when the config is not present?

Say, when the GCP bucket is missing, we could log as a warning that Downloads, Regression tests, etc. won't work as expected.

Reply:
Yes, exactly; we could log functionalities that might not work corresponding to each missing entry. We can create a GitHub issue for the same for now.

Reply:
Cool. Please create an issue, and add a reference to this conversation. I'm merging the PR, thanks for the great work.

Reply:
Please mark the conversation resolved once you have created the issue.